### PR TITLE
Add Memcached CI

### PR DESCRIPTION
**Pull request by rullzer**
TODO:
- [x] Enable all other tests again
- [x] Fix memcached errors

### CODE REVIEW

File: lib/private/Memcache/Memcached.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ public function __construct($prefix = '') {
 				\Memcached::OPT_LIBKETAMA_COMPATIBLE => true,
 
 				// Enable Binary Protocol
-				\Memcached::OPT_BINARY_PROTOCOL =>      true,
+				//\Memcached::OPT_BINARY_PROTOCOL =>      true,
```
Comment (rullzer) [inline lib/private/Memcache/Memcached.php]: Enabling this made the tests fail

Comment (rullzer) [inline lib/private/Memcache/Memcached.php]: uhm it kind of works. but when adding a key that already exists I get the wrong error code. This is on debian so they could just be shipping ancient stuff ( :speak_no_evil: )

**Comment by MorrisJobke** [comment]
@rullzer: okay :) could you fix the conflicts?

File: lib/private/Memcache/Memcached.php (hunk 1 above)
Comment (rullzer) [inline lib/private/Memcache/Memcached.php]: Mmmmm ok so even locally on a never memcached etc the same tests fail. So disabling the binary protocol seems best for now

**Comment by rullzer** [comment]
Rebased

**Review by MorrisJobke — APPROVED** [review]

**Review by LukasReschke — APPROVED** [review]

**Pull request merged**